group_vars/all.yml.sample

```diff
@@ -690,8 +690,8 @@ dummy:
 # By default, keys will be auto-generated.
 #
 #openstack_keys:
-#  - { name: client.glance, caps: { mon: "profile rbd", osd: "profile rbd pool=volumes, profile rbd pool={{ openstack_glance_pool.name }}, osd blacklist"}, mode: "0600" }
```

Comment:
the glance key does not need r/w on the volume pools, this is correct

Reply:
I might have added this comment in the wrong place, I meant that it is correct to remove permissions on the volumes from the glance key